Cargo.toml

```diff
@@ -1,33 +1,32 @@
 [package]
-authors = ["Kjetil Kjeka <kjetilkjeka@gmail.com>"]
+authors = ["Kjetil Kjeka <kjetilkjeka@gmail.com>", "Tmplt <tmplt@dragons.rocks>"]
 categories = ["embedded", "no-std", "hardware-support"]
 repository = "https://github.com/kjetilkjeka/s32k144evb.rs"
 description = "Board support crate for s32k144evb"
 keywords = ["arm", "cortex-m", "s32k144", "template"]
 license = "MIT OR Apache-2.0"
 name = "s32k144evb"
-version = "0.6.1"
+edition = "2018"
+version = "0.7.0"
```

Comment:
Please hold on upping this version, there might be a few more things going into this release.

Reply:
Reverted back to v0.6.1.